### PR TITLE
Choose annotations

### PR DESCRIPTION
**Pull request by LBoleij**

### CODE REVIEW

**Comment by Jipje** [comment]
Makes a new window that can only be accessed when annotations are opened.
Besides that it colours the annotations.

File: src/main/java/graph/SequenceNode.java [hunk 1]
```diff
@@ -100,15 +100,51 @@ public void draw(GraphicsContext gc, ColourController colourController) {
...
+            
+           colourMeBby = colourController.getColors(genomes);
             double tempCoordinate = yCoordinate;
             double tempHeight = height / colourMeBby.size();
             for (Color beamColour : colourMeBby) {
                 gc.setFill(beamColour);
                 gc.fillRect(xCoordinate, tempCoordinate, width, tempHeight);
                 tempCoordinate += tempHeight;
             }
+            for (int i = 0; i < annotations.size(); i++) {
```
Comment (JaspervanTilburg) [inline src/main/java/graph/SequenceNode.java]: Maybe split up in smaller methods?

File: src/main/java/gui/MenuController.java [hunk 2]
```diff
@@ -517,4 +521,44 @@ private void genomeBookmarkClicked(MenuItem bookmark) {
             GraphDrawer.getInstance().redraw();
         }
     }
+
+    /**
+     * Handles choosing Annotations.
+     * @throws IOException if something goes wrong.
+     */
+    public void chooseAnnoClicked() throws IOException {
+        FXMLLoader loader = new FXMLLoader(getClass().getResource("/FXML/AnnotationTable.fxml"));
```
Comment (JaspervanTilburg) [inline src/main/java/gui/MenuController.java]: Nice job!

File: src/main/java/gui/sub_controllers/AnnotationTableController.java [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+        allSelected = false;
+        selection = new ArrayList<Annotation>();
+        masterData = FXCollections.observableArrayList(annotations);
+
+        // 0. Initialize the columns.
+        idColumn.setCellValueFactory(new PropertyValueFactory<Annotation, Integer>("id"));
+        startColumn.setCellValueFactory(new PropertyValueFactory<Annotation, Integer>("start"));
+        endColumn.setCellValueFactory(new PropertyValueFactory<Annotation, Integer>("end"));
+        infoColumn.setCellValueFactory(new PropertyValueFactory<Annotation, String>("info"));
+        highlightColumn.setCellValueFactory(
```
Comment (JaspervanTilburg) [inline src/main/java/gui/sub_controllers/AnnotationTableController.java]: Your welcome ;)

**Pull request merged**